### ISSUE TITLE
States palette: click on tooltip "state" => tooltip doesn't show

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1443]
- New HTML File (flow layout)
- Add a Button
- Drop a Tooltip on Button
- Click on Button to select
- Click on Tooltip in Outline palette => Tooltip shows and gets selected (correct!)
- Click on Button to select => Tooltip disappears
- In States palette, click on "_show_Tooltip..."

Result => Tooltip doesn't show

**Comment by peller** [comment]
Yeah, used to work, but I think that code is in limbo pending a better states solution for overlay and mobile.

**Comment by jhpedemonte** [comment]
-> M6

**Comment by peller** [comment]
-> M7
